velox/exec/fuzzer/WindowFuzzer.h

```diff
@@ -95,9 +95,9 @@ class WindowFuzzer : public AggregationFuzzerBase {
 
   void addWindowFunctionSignatures(const WindowFunctionMap& signatureMap);
 
-  const std::string generateFrameClause();
+  std::tuple<std::string, bool> generateFrameClause();
```

Comment:
Could you add a comment to explain what's in the tuple, i.e. what do the string and particularly the bool represent, the latter isn't clear from the function name.

Reply:
Oops, forgot to add that comment. Thanks for catching it!